samples/helloworld_esm/worker.js

```diff
@@ -2,8 +2,17 @@
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
 
+addEventListener('error', (event) => {
+  console.log('reported error:', event.error);
+//  event.preventDefault();
+});
+
+reportError(new DOMException('boom'));
+
 export default {
   async fetch(req, env) {
+    reportError(new DOMException('boom'));
+
```

Comment:
The changes in this file are temporary... being used for testing... proper tests and sample files will be added.